### DISCUSSION TITLE
Which linux-surface image maps to which Ubuntu Kernel

### DISCUSSION THREAD

**Discussion by istokes1** [linux-surface/linux-surface#1370] [Q&A]
I initially installed Ubuntu using the 6.5 kernel, but later added the Surface image, which upgraded me to the 6.7 version. Due to compatibility issues with the base 6.5 kernel, I had to downgrade to the 5.15.0-94 kernel to use other applications. Now, I'm trying to determine which version of the Surface image I can add back to ensure the Surface functionality works. It needs to be based on the core 5.15.0-94 kernel or something closely compatible, I believe. Please can you advise me.

**Comment by qzed** [comment]
Unfortunately, we only provide a single kernel version across all distributions that we support. We just don't have the time to maintain multiple versions right now, sorry. And most of the time, things should work fine even with a different kernel version than the distro default.

If you do happen to run into these problems, however, I would recommend switching to a different distribution that uses newer kernels. For example Fedora. For both Fedora and Arch, we are (normally) in sync.
